spec/system/user_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'user pages' do
+  let(:membership) { create :membership, admin: true}
+  let!(:admin) { create :user, first_name: 'tester', memberships: [membership] }
+
+  context 'viewing the index' do
+    it 'directs you to the appropriate page' do
+      set_current_user(admin)
```

Comment:
Uhhhh should this sample test be involved? (AKA should I review it?)

Reply:
No, don't worry about it. This PR gets system tests set up and makes sure they work, this test should later actually be fleshed out but not right now.